src/mappings/discord/utils/ComponentDispatchUtils.ts

```diff
@@ -0,0 +1,24 @@
+import { ModuleExportType } from "@moonlight-mod/moonmap";
+import register from "../../../registry";
+
+register((moonmap) => {
+  const name = "discord/utils/ComponentDispatchUtils";
+  moonmap.register({
+    name,
+    find: "ComponentDispatchUtils",
+    process({ id }) {
+      moonmap.addModule(id, name);
+
+      moonmap.addExport("discord/utils/ComponentDispatchUtils", "ComponentDispatcher", {
```

Comment:
Nitpick: I'd pass `name` as the first argument here in case we need to rename it.